msg/battery_status.msg

```diff
@@ -18,6 +18,8 @@ uint16 run_time_to_empty   # predicted remaining battery capacity based on the p
 uint16 average_time_to_empty   # predicted remaining battery capacity based on the average rate of discharge in min
 uint16 serial_number       # serial number of the battery pack
 
+float32[4] voltage_cell_v   # Battery individual cell voltages
```

Comment:
This message is getting pretty badly overloaded. It started out as the status of an analog current and voltage sensor.

Reply:
What would you like to see? Break it out into two messages?

Reply:
It's fine in this PR, but it's yet another sign that the entire battery and system power architecture isn't quite right. We have different fields mashed together for different sources and no way of knowing which fields are expected to be there. For downstream modules to depend on this information (eg commander trigger a battery failsafe) it needs to be treated more like an interface.

Ideas for later - 

We should be able to support a variable number of batteries from any source (analog, smbus, mavlink).

 - battery_status (many)
   - analog (most pixhawks have > 1 now, but only parameters for the first)
   - smbus (should be able to have as many of this as you want)
   - mavlink
 - system_power
   - 5v and 3.3v status
   - usb power valid
   - servo rail voltage
   - number of analog bricks connected

All of this should probably be rolled together into a power module where we can configure what's available on the actual system.